0010-zkapp-cli-lightnet-management.md

```diff
@@ -0,0 +1,100 @@
+# RFC for zkapp-cli enhancements to provide the local lightnet networks management
+
+Introduce new sub-commands to the existing zkapp CLI application to streamline the local lightnet networks management for development and testing purposes.
+
+## Summary
+
+We aim to extend the current zkapp CLI application by introducing the `lightnet` sub-command suite. This suite will centralize and simplify Docker containers and provided tools management tasks for local zkApps development and testing, enhancing user and developer experience.
+
+## Motivation
+
+The primary motivation is to simplify the local zkApps development and testing process by offering intuitive CLI commands.
+
+Expected Outcomes:
+
+- A streamlined workflow for managing local lightnet network [Docker containers](https://hub.docker.com/r/o1labs/mina-local-network) and tools provided by such containers.
+- Reduced complexity and better DX, eliminating manual 3rd party tools management.
+
+## Detailed design
+
+### zkapp CLI Interface Enhancements
+
+Under the primary command `zk`, the new `lightnet` sub-commands will be introduced:
+
+#### Help
+
+- **Command**: `zk lightnet`
+- **Output**: Help message with description of available sub-commands and their options.
```

Comment:
I like this pattern to list the help message with sub commands after entering `zk lightnet`. We should also add `zk lightnet ` and the subcommands to the help message a user can invoke when entering 'zk' or 'zk --help'. It can look something like this.

Usage: zk <command> [options]
```

Commands:
  zk project [name]  Create a new project                     [aliases: proj, p]
  zk file [name]     Create a file and generate the corresponding test file
                                                                    [aliases: f]
  zk config          Add a new deploy alias
  zk deploy [alias]  Deploy or redeploy a zkApp
  zk example [name]  Create an example project                      [aliases: e]
  zk system          Show system info                          [aliases: sys, s]
  zk lightnet [sub-command]
        start <options>       Start  a light network              [aliases: tbd]
           List start options
        stop <options>        Stop  a light network               [aliases: tbd]
            List stop options
        logs  <sub-command>       Display network logs            [aliases: tbd]
            List logs sub-commands
        explorer     Start  a explorer                            [aliases: tbd]
Options:
  -h, --help     Show help                                             [boolean]
  -v, --version  Show version number                                   [boolean]

```

Reply:
Thank you @ymekuria, that was the idea.